ios/PagerViewProvider.swift

```diff
@@ -0,0 +1,113 @@
+import SwiftUI
+import UIKit
+
+@objc public enum KeyboardDismissMode: Int {
+  case none
+  case onDrag
+  
+  func convert() -> UIScrollView.KeyboardDismissMode {
```

Comment:
Personal preference: use computed vars when their are no arguments to the function